### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by t-my** [deurk/ktx#15]
Is there any type of documentation around?

Trying to setup some coffee break madness for my company. Would like to serve some midair, instagib and so on but there is a great void how to configure msdsv/ktx. 

Thanks!

**Comment by niclaslindstedt** [comment]
There's a wiki on this GitHub page that anyone can update, so it would be nice if we could all come together and try to get some basic documentation up.

**Comment by jite** [comment]
I totally agree. Question is though if we should document it here or on quakeworld.nu wiki (that will undergo an upgrade in the near future). Since these projects are rather inactive I think having everything on one place might be more suitable, what do you think? (In general I think that docs should be as close to the src as possible and external parties to link to it, but this is a bit different due to status)

**Comment by niclaslindstedt** [comment]
I think the documentation should be here on GitHub.

If we try to keep the pages to a minimum, it can easily be duplicated to QW.nu wiki.

**Comment by jite** [comment]
No please no duplication, that's the worst thing to do. As soon as something changes at one place there are two (or more) sources of information that might contradict eachother + it's extra maintenance. The reasonable options IMO are to either host it on github and link from qwnu wiki or host on qwnu wiki and link to it from github.

**Comment by niclaslindstedt** [comment]
Then link it here, imo.

**Comment by jite** [comment]
Why do you think so? :) My arguments for having docs on wiki:
* I imagine it's easier for people to find
* The projects have a tight connection and usually work together so having the documentation in one place would simplify compared to per github project page, since that would mean atleast 4 different places for each module.
* Repo is rather unmaintained/inactive, it's all up to deurks control and might dissappear anytime (like qw-dev)

**Comment by niclaslindstedt** [comment]
The inactivity bit is a good argument, so sure. I agree.

**Comment by tsoikkel** [comment]
+1 for version controlled documentation. Please don't do it in forums or anywhere else. GitHub pages is great tool and people can fork and continue working if one project decides to quit.

**Comment by jite** [comment]
@tsoikkel That's the best in the generic case yes but not necessarily in this case. The qw scene is rather focused around quakeworld.nu and ktx/mvdsv aren't used without eachother in 99% of the cases for instance. The quakeworld wiki is "version" controlled in the sense that it's a mediawiki.

**Comment by tsoikkel** [comment]
Sorry I haven't been following the community for 10 years. Now just popped in and tried to setup a server.

Ended up into:

KTX: http://www.quakeworld.nu/forum/topic/2504/ktx-kombat-teams-extreme-files
MVDSV: http://www.quakeworld.nu/forum/topic/2505/mvdsv-multiview-demo-server-files

This is exactly the case when personally hosted services are huge risk to actually losing data.

**Comment by deurk** [comment]
Documentation will be hosted on Github. I will start writing it, any help is of course welcome towards this community effort. It's about time this project gets a decent documentation.

**Comment by deurk** [comment]
Putting that here to not forget to add it to documentation:
https://gist.github.com/meag/09b7b6c736b8f2fde1df991cf1151abb
https://gist.github.com/meag/b5f3e37777d4e3233cba5df2046d1323
